avocado/core/nrunner_avocado_instrumented.py

```diff
@@ -20,6 +20,7 @@ class and method names should be separated by a ":".  One
 
      * args: not used
     """
+    DEFAULT_TIMEOUT = 86400
```

Comment:
How could we make this a custom setting by the users? This is something that I'm trying to do for the "skip exit codes".